Dockerfile

```diff
@@ -28,6 +28,8 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
     tzdata \
     gpg \
     apt-utils \
+    libc6-dev \
+    gcc \
```

Comment:
C compiler is required to run a race test on non-darwin, see requirements: https://go.dev/doc/articles/race_detector#Requirements